src/server.c

```diff
@@ -5303,23 +5303,23 @@ void commandGetKeysCommand(client *c) {
 void commandHelpCommand(client *c) {
     const char *help[] = {
 "(no subcommand)",
-"    Return details about all Redis commands.",
+"    Return details about all Valkey commands.",
```

Comment:
Let's do help commands in a separate PR too. I did this without seeing your change: #216